Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM unocha/nodejs:8.11.3
+
+ENV CTF_SPACE_ID=0123456789ab \
+    CTF_CDA_ACCESS_TOKEN=0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqr
```

Comment:
same here, these tokens are just made up.

Reply:
These ones are inserted in the docker image when it is built.
We can guarantee in theis way that the two env vars will be available in a running container no matter how you start that container.